src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2831,7 +2831,7 @@ internal static bool QueryServiceConfig(NakedWin32Handle hService, out NativeMet
                 cbBufSize: 0,
                 pcbBytesNeeded: out bufferSizeNeeded);
 
-            if (status != true && Marshal.GetLastWin32Error() != ERROR_INSUFFICIENT_BUFFER)
```

Comment:
This also isn't code redundancy, please try to fix one issue in a PR, and limit the size of a PR.

Reply:
http://dev2ops.org/2012/03/devops-lessons-from-lean-small-batches-improve-flow/
https://cloud.google.com/solutions/devops/devops-process-working-in-small-batches

Reply:
It is a redundant boolean literal. I can split into seperate PR.

Reply:
@xtqqczze In future it is better to make one fix per a PR but current I reviewed the PR twice :-) so please do not break a commit history.

Reply:
@TravisEz13 @iSazonov I was thinking about seperating 02f4b96 into a new PR, but I would rather not break the commit history again.